app/server/api/disbursement_api.py

```diff
@@ -217,9 +216,12 @@ def put(self, disbursement_id):
                 disbursement.approve()
                 db.session.commit()
                 auto_resolve = AccessControl.has_sufficient_tier(g.user.roles, 'ADMIN', 'superadmin')
```

Comment:
I think this is the guilty line.

Reply:
That makes sense. The original spec was that anyone could draft a bulk disbursement, but a superadmin had to actually make the transactions. I'm not entirely happy with this solution, but I changed it to `if g.active_organisation.require_multiple_transfer_approvals or AccessControl.has_sufficient_tier(g.user.roles, 'ADMIN', 'superadmin')`, just because when requiring multiple approvers is disabled we probably still want to limit disbursement ability to superadmin.